src/index.tsx

```diff
@@ -12,10 +12,12 @@ import AppConfig from './domain/app/AppConfig';
 initI18next();
 Modal.setAppElement('#root');
 
-if (import.meta.env.VITE_ENVIRONMENT === 'production') {
+console.warn('Environment prod: ', import.meta.env.PROD);
+
+if (import.meta.env.PROD) {
   Sentry.init({
     dsn: import.meta.env.VITE_SENTRY_DSN,
-    environment: import.meta.env.VITE_ENVIRONMENT,
+    environment: import.meta.env.PROD ? 'production' : 'development',
```

Comment:
**This change should be rejected!** Environment was set properly: It would be clear in the Sentry that it was the review-server or the test server that sent the error. No need to change this! This `import.meta.env.VITE_ENVIRONMENT` is set to "review" in development pipelines, meaning the PR-servers. It's set to testing, staging and production in the other pipelines that we have.

Reply:
the environment option in Sentry init configuration is to define a request parameter when the log is created in sentry. It defines the environment (review, testing, staging, production).

The Sentry should be initialized only when a built & bundled version is running (`import.meta.env.PROD === true` or `NODE_ENV === 'production'`)